prisma/schema.prisma

```diff
@@ -239,24 +252,25 @@ model DepositHead {
 }
 
 model MaspTransaction {
-  id               Int                 @id @default(autoincrement())
-  created_at       DateTime            @default(now()) @db.Timestamp(6)
-  updated_at       DateTime            @default(now()) @updatedAt @db.Timestamp(6)
-  transaction_hash String              @db.VarChar
-  block_hash       String              @db.VarChar
-  asset_identifier String              @db.VarChar
-  type             MaspTransactionType
+  id               Int       @id @default(autoincrement())
+  created_at       DateTime  @default(now()) @db.Timestamp(6)
+  updated_at       DateTime  @default(now()) @updatedAt @db.Timestamp(6)
+  transaction_hash String    @db.VarChar
+  block_hash       String    @db.VarChar
+  asset_name       String    @db.VarChar
+  type             EventType
```

Comment:
Jason suggested we include the type here so that if the events get deleted we will still have the record. He also said just to use the `EventType` enum and we can only ever use the `MASP*` for these entries.